wpilibj/src/main/java/edu/wpi/first/wpilibj/ADIS16470_IMU.java

```diff
@@ -935,57 +779,22 @@ private void acquire() {
             m_accel_x = accel_x;
             m_accel_y = accel_y;
             m_accel_z = accel_z;
-            m_compAngleX = compAngleX * rad_to_deg;
-            m_compAngleY = compAngleY * rad_to_deg;
-            m_accelAngleX = accelAngleX * rad_to_deg;
-            m_accelAngleY = accelAngleY * rad_to_deg;
-            m_average_gyro_rate_x += gyro_rate_x;
-            m_average_gyro_rate_y += gyro_rate_y;
-            m_average_gyro_rate_z += gyro_rate_z;
```

Comment:
Where did this code go?

Reply:
The average variables were unused, so I removed them. They don't exist in C++.